Index: startScreen.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><html lang=\"en\">\r\n<head>\r\n\t<meta charset=\"UTF-8\">\r\n\t<meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\r\n\t<title>Startscherm</title>\r\n\t<link rel=\"stylesheet\" href=\"style.css\">\r\n</head>\r\n\r\n<body class=\"body\">\r\n\r\n<h1 class=\"yes\" >The Best Seller</h1>\r\n\r\n<input type=\"button\" class=\"button\" onclick=\"location.href='joinGameScreen.html'\" value=\"Join Game\">\r\n<input type=\"button\" class=\"button\" onclick=\"location.href='CreateGameScreen.html'\" value=\"Create Game\">\r\n\r\n<div class=\"content\">Click a button to change the screen.</div>\r\n\r\n\r\n</body>\r\n</html>
===================================================================
diff --git a/startScreen.html b/startScreen.html
--- a/startScreen.html	
+++ b/startScreen.html	
@@ -10,7 +10,7 @@
 
 <h1 class="yes" >The Best Seller</h1>
 
-<input type="button" class="button" onclick="location.href='joinGameScreen.html'" value="Join Game">
+<input type="button" class="button" onclick="location.href='index.html'" value="Join Game">
 <input type="button" class="button" onclick="location.href='CreateGameScreen.html'" value="Create Game">
 
 <div class="content">Click a button to change the screen.</div>
